cram_common/cram_manipulation_interfaces/src/package.lisp

```diff
@@ -42,6 +42,8 @@
    #:object-type-subtype
    #:object-type-direct-subtype
    #:robot-free-hand
+   #:unidirectional-attachment
```

Comment:
Need an empty definition of this fact in man-int with `` (fail) `` as implementation.

Reply:
Done. https://github.com/cram2/cram/pull/112/commits/aca67bc0405d64b692c3cef7ed6479557f29c946